lib/config.h

```diff
@@ -79,8 +77,7 @@
 
 // unused
 #if defined(__GNUC__) \
-    || defined(__clang__) \
-    || defined(__CPPCHECK__)
+    || defined(__clang__)
```

Comment:
How will it work when I run selfcheck manually on the command line? Before I only had to provide `-D__CPPCHECK__` to get reasonable results. It seems tedious to provide all gnu preprocessor flags manually.

I run cppcheck selfcheck manually sometimes..